modular-docs-manual/content/topics/assembly_mod-docs-conversion.adoc

```diff
@@ -0,0 +1,38 @@
+[[converting-to-mod-doc]]
+= Converting to Modular Documentation
+
+If you have a monolithic, feature-based manual, you can convert it to a set of modular content based on user stories. This conversion workflow involves using the customer product lifecycle to define user stories for your product, and creating the assemblies and modules necessary to fit each user story.
+
+The result is documentation that is more relevant for your readers, because it is based on real-world user stories (it tells them how to accomplish their goals), and modular (it can be assembled into whatever sets and formats they might need). 
+
+
+== Prerequisites
+
+* You should understand the following important concepts:
+** xref:what-modular-documentation-is[What Modular Documentation Is]
+** xref:what-modular-documentation-is-not[What Modular Documentation Is Not]
+** Modular documentation benefits [NEED LINK]
+** xref:modular-docs-terms-definitions[Modular Documentation Terminology]
+
+* Find a user story and practice documenting it as an xref:assembly-guidelines[assembly].
++
+This process of breaking down user stories into assemblies and modules is an essential element of modular documentation. You should be familiar with this process before attempting to do it for an entire feature-based book.
+
+* Your repository should be set up.
++
+--
+If you are using AsciiDoc, your repository should have a directory structure that supports modular documentation, and you should have a `master.adoc` file for each publication.
+
+[NEED LINK] 
```

Comment:
Ben, would it be useful to include before and after examples of how a repo is restructured for modular? If you recall from a few months ago, the workflow I wrote up had information about this. It might be useful to revisit for this workflow you've created. [Modular conversion workflow](https://docs.google.com/a/redhat.com/document/d/1Sub2Mp8FqBN2io0E8R_d6n0sAJhDzEABV0zrJ2Ss_6w/edit?usp=sharing)